app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -489,10 +489,7 @@ private void initPlayer(final boolean playOnReady) {
         simpleExoPlayer.addTextOutput(binding.subtitleView);
 
         // Setup audio session with onboard equalizer
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-            trackSelector.setParameters(trackSelector.buildUponParameters()
-                    .setTunnelingAudioSessionId(C.generateAudioSessionIdV21(context)));
-        }
+        trackSelector.setParameters(trackSelector.buildUponParameters().setTunnelingEnabled(true));
```

Comment:
Is that backwards compatible for NewPipe legacy (Android 4.1) or do we need to notify friendlyanon about this change?

Reply:
Yes, it's safe. In ExoPlayer source they have basically the same SDK version check before either applying generateAudioSessrionIdV21 or setting it to UNSET on lower Api

Reply:
https://github.com/google/ExoPlayer/blob/4364b9150f236813a2070fc79d3bcb76e11969ac/library/core/src/main/java/com/google/android/exoplayer2/SimpleExoPlayer.java#L662

And in the renderers there are also V21 methods regarding tunneling. 

From the release notes:
Simplify enabling tunneling with DefaultTrackSelector. ParametersBuilder.setTunnelingAudioSessionId has been replaced with ParametersBuilder.setTunnelingEnabled. The player's audio session ID will be used, and so a tunneling specific ID is no longer needed.